File: fizzbuzz.py
def fizzbuzz(low, high):
    for num in range(low, high):
        temp = ""
        if num % 3 == 0:
            temp += 'Fizz'
        if num % 5 == 0:
            temp += 'Buzz'
        if temp == "":
            temp = num
        print(temp)

# user input for low end of the range
low = int(input("Min: "))

# user input for high end of the range
high = int(input("Max: "))

fizzbuzz(low, high)

File: daily-rb/fizzbuzz.rb
def fizzbuzz(limit)
  threes = []
  fives = []
  (1..limit).each do |num|
    threes << num * 3 if num * 3 <= limit
    fives << num * 5 if num * 5 <= limit
  end
  (1..limit).each do |num|
    puts num unless (threes | fives).include?(num)
    puts 'fizz' if (threes - fives).include?(num)
    puts 'buzz' if (fives - threes).include?(num)
    puts 'fizzbuzz' if (threes & fives).include?(num)
  end
end

fizzbuzz(100)
